pkg/reconciler/chart/component.go

```diff
@@ -15,6 +15,10 @@ type Component struct {
 	configuration map[string]interface{}
 }
 
+func (c *Component) isExternalComponent() bool {
```

Comment:
this is used to figure out path of the `Chart.yaml` for external components. The reason is that native kyma components have different pathing structure and for those, the hardcoded path is fine
```
/tmp/reconciler/2.0.0/[component_name]/Chart.yaml
```
for external components, we don't always have full control over how it's packed, the `Chart.yaml` might be at root level or nested few folders deep, so the hardcoded path may not suffice
```
/tmp/reconciler/[hash]-[component_name]/[component_name]/Chart.yaml
```
Instead for those I would like to propose recursive search for `Chart.yaml` starting at
```
/tmp/reconciler/[hash]-[component_name]/
```